CMakeLists.txt

```diff
@@ -91,8 +91,8 @@ include_directories(${FSLAM_BINARY_DIR}/cpp)
 add_subdirectory(cpp/core)
 add_subdirectory(cpp/fastslam1)
 add_subdirectory(cpp/fastslam2)
-enable_testing()
-add_subdirectory(cpp/gtest)
+# enable_testing()
+# add_subdirectory(cpp/gtest)
```

Comment:
Can you reset this to include the tests again? I don't see a reason why they should be excluded.

Reply:
Done.